src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -195,7 +197,7 @@ internal static bool IsInbox
         /// Some common environment variables used in PS have different
         /// names in different OS platforms
         /// </summary>
-        internal static class CommonEnvVariableNames
+        public static class CommonEnvVariableNames
```

Comment:
I was thinking it would be useful for cmdlet authors rather than having to rely on `HOME` for Unix and `USERPROFILE` for Windows